wallet_transactions/transactions/permission.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import permissions
+
+
+class IsOwnerTransaction(permissions.IsAuthenticated):
+    """check permissions is this user is owner"""
+
+    def has_permission(self, request, view):
+        if request.user.is_authenticated:
+            return True
+        return False
+
+    def has_object_permission(self, request, view, obj):
+        if obj.sender.owner == request.user or obj.receiver.owner == request.user:
+            return True
+        return False
```

Comment:
please check if it works for logged in user.
does a user get only his wallets?
later this permission need to tested with transactions

Reply:
Yes, it is work